src/m365/spo/commands/folder/folder-rename.ts

```diff
@@ -78,7 +82,7 @@ class SpoFolderRenameCommand extends SpoCommand {
         data: `<Request AddExpandoFieldTypeSuffix="true" SchemaVersion="15.0.0.0" LibraryVersion="16.0.0.0" ApplicationName="${config.applicationName}" xmlns="http://schemas.microsoft.com/sharepoint/clientquery/2009"><Actions><Method Name="MoveTo" Id="32" ObjectPathId="26"><Parameters><Parameter Type="String">${renamedServerRelativeUrl}</Parameter></Parameters></Method></Actions><ObjectPaths><Identity Id="26" Name="${folderObjectIdentity.objectIdentity}" /></ObjectPaths></Request>`
       };
 
-      
+
       const res = await request.post<any>(requestOptions);
```

Comment:
Here, let's change `.post<any>` to `post<string>` as that's the value type we're expecting.